packages/vite/src/node/plugins/css.ts

```diff
@@ -1771,6 +1814,7 @@ async function minifyCSS(
   css: string,
   config: ResolvedConfig,
   inlined: boolean,
```

Comment:
```suggestion
  inlined: boolean,
  filename: string,
```
Let's pass a filename so that we can pass it to lightningcss/esbuild when the input is a non-concatenated file.

Reply:
I set the `filename: string | undefined` since there were some places, like `extractCss` there were no appropriate filename to pass down.